src/Avalonia.Ide.CompletionEngine/AssemblyMetadata/MetadataConverter.cs

```diff
@@ -64,6 +84,26 @@ public static Metadata ConvertMetadata(IMetadataReaderSession provider)
                         foreach (var alias in nsAliases) metadata.AddType(alias, mt);
                 }
 
+                const string avaresToken = "Build:"; //or "Populate:" should work both ways
```

Comment:
`Populate` will probably not available for styles with x:Class inside. So `Build` is correct